web-console/src/lib/components/connectors/DataSourceTable.tsx

```diff
@@ -188,7 +214,12 @@ const DataSourceTable = () => {
           toolbarChildren={[
             btnAdd,
             <GridToolbarFilterButton key='1' />,
-            <div style={{ marginLeft: 'auto' }} key='2' />
+            <ResetColumnViewButton
+              key='2'
```

Comment:
why 2?

Reply:
`key` is a value required to distinguish React components in a list; arbitrarily, btnAdd has key='0', and so on